src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -59,22 +59,17 @@ const ProgressiveProfiling = (props) => {
     ensureAuthenticatedUser(DASHBOARD_URL)
       .then(() => {
         hydrateAuthenticatedUser().then(() => {
+          identifyAuthenticatedUser(authenticatedUser?.userId);
           setReady(true);
         });
       })
       .catch(() => {});
```

Comment:
Why do we have catch block? I think we should remove it.

Reply:
It was already there, I haven't added it.

Reply:
make sense i thought you can remove it

Reply:
@ahtesham-quraish, if you look at the implementation of `ensureAuthenticatedUser` [here](https://github.com/openedx/frontend-platform/blob/afe3a1aa894aad413c97c315e157559d7f84be43/src/auth/AxiosJwtAuthService.js#L266-L270).

You will see that it works in a way that if the user is not authenticated, then redirect the user to the login page and throw an error along with it. This error was harmless but a useless noise in the logs as well. So we just added a catch block to don't throw errors on the console when the such scenario happens.